pinodo/coffee-mathcer
# feat: Make an algorithm to recommend a coffee based on mood and weather

### Summary
  1. Implemented an algorithm to calculate the best coffee based on moods users have selected and the current weather.

### Steps to test 

Add the following code snippet to `App.jsx` and see that you get the correct coffee recommendation when you change `temperature` and `selectedMoodIDs` values

 ```
import recommendCoffee from './lib/coffeeRecommendationAlgorithm'

const selectedMoodIDs = [0, 2]
  const temperature = 19
  const coffee = recommendCoffee(selectedMoodIDs, temperature)
  console.log(coffee)
```
